includes/Core/Modules/Modules.php

```diff
@@ -1077,6 +1079,62 @@ private function get_rest_routes() {
 					),
 				)
 			),
+			new REST_Route(
+				'core/modules/recover-module',
+				array(
+					array(
+						'methods'             => WP_REST_Server::EDITABLE,
+						'callback'            => function( WP_REST_Request $request ) {
+							$data = $request['data'];
+							$slug = isset( $data['slug'] ) ? $data['slug'] : '';
+							try {
+								$module = $this->get_module( $slug );
+							} catch ( Exception $e ) {
+								return new WP_Error( 'invalid_module_slug', $e->getMessage(), array( 'status' => 404 ) );
+							}
+
+							if ( ! $module->is_shareable() ) {
+								return new WP_Error( 'module_not_shareable', __( 'Module is not shareable.', 'google-site-kit' ), array( 'status' => 404 ) );
+							}
+
+							if ( ! $this->is_module_recoverable( $module ) ) {
+								return new WP_Error( 'module_not_recoverable', __( 'Module is not recoverable.', 'google-site-kit' ), array( 'status' => 403 ) );
+							}
+
+							$check_access_endpoint = '/' . REST_Routes::REST_ROOT . '/' . self::REST_ENDPOINT_CHECK_ACCESS;
+							$check_access_request = new WP_REST_Request( 'POST', $check_access_endpoint );
+							$check_access_request->set_body_params(
+								array(
+									'data' => array(
+										'slug' => 'analytics',
+									),
+								)
+							);
```

Comment:
Why is the `analytics` slug hardcoded here? Seems like this might be a debugging statement accidentally left in 😅

Reply:
Unbelievable 🙄 😄